app/components/Feedback/Form.tsx

```diff
@@ -27,6 +28,9 @@ const FeedbackForm = ({onSubmit, onClose, options, className}: FeedbackFormProps
 
   return (
     <div ref={clickCheckerRef} className={'feedback-form bordered ' + (className ?? '')}>
+      <button onClick={onClose} className="mobile-only escape-feedback">
+        <XLarge></XLarge>
```

Comment:
`<XLarge />` is how it's done when the tag doesn't have any contents

Reply:
too true